src/store/modules/bot-commands/bot-commands-mutations.ts

```diff
@@ -1,14 +1,27 @@
 import { MutationTree } from 'vuex'
 import { BotCommand, BotCommandsState } from '@/store/modules/bot-commands/types.ts'
+import { uniq } from 'lodash'
 
 export const mutations: MutationTree<BotCommandsState> = {
   addCommand(state, botCommand: BotCommand): void {
     const commandValue = botCommand.command.trim()
-    const botCommands = state.commands[botCommand.partnerId]
+    let botCommands = state.commands[botCommand.partnerId]
     if (!botCommands) {
-      state.commands[botCommand.partnerId] = [commandValue]
-    } else if (!botCommands.includes(commandValue)) {
+      botCommands = [commandValue]
+    } else {
+      const index = botCommands.indexOf(commandValue)
+      if (index >= 0) {
+        botCommands.splice(index, 1)
+      }
       botCommands.push(commandValue)
     }
+    state.commands[botCommand.partnerId] = botCommands
+  },
+
+  initCommands(state, { partnerId, commands }: { partnerId: string; commands: string[] }): void {
+    const botCommands = state.commands[partnerId]
+    if (!botCommands) {
```

Comment:
I think you can skip this check and override the commands anyway.

Scenario:

1. You opened a chat in PWA app
2. Initial bot commands were saved to Session Storage
3. From iOS app you send new commands
4. Back to PWA. Refresh the page

**Expected result:** Init commands again

**Actual result:** Initialization is skipped. Using old state from SS